http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterToolMonitor.scala

```diff
@@ -1,88 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo
-package monitor
-
-import akka.actor.{Actor, Props, Timers}
-import cats.effect.IO
-import cats.effect.unsafe.implicits.global
-import cats.syntax.all._
-import com.typesafe.scalalogging.LazyLogging
-import org.broadinstitute.dsde.workbench.leonardo.config.ClusterToolConfig
-import org.broadinstitute.dsde.workbench.leonardo.dao.ToolDAO
-import org.broadinstitute.dsde.workbench.leonardo.db.{clusterQuery, DbReference}
-import org.broadinstitute.dsde.workbench.leonardo.monitor.ClusterToolMonitor._
-import org.broadinstitute.dsde.workbench.openTelemetry.OpenTelemetryMetrics
-
-import scala.concurrent.ExecutionContext
-
-object ClusterToolMonitor {
```

Comment:
As mentioned in the description, this class had a similar intent to `LeoMetricsMonitor` but (a) it was checking GCP/Jupyter only; and (b) I don't think it was actually running. :) 

I also was surprised to see an akka `Actor`… I thought we removed them all in favor of `fs2`.